### PR TITLE
Styling done-Prerna

### PR DESCRIPTION
**Pull request by Prerna-Goel**
task3 done!!

#50

### CODE REVIEW

**Comment by lisa761** [comment]
Great work! If you are sure you do not want to add anything, should I merge this?

File: participants/Prerna-Goel.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+
+<h1><u><i>Hey!! This is Prerna Goel </h1>
+<b>GitHub username: </b> <a href= "https://github.com/Prerna-Goel/" >Prerna-Goel</a>
+<hr>
+<b> University: </b> <a href = "[https://www.igdtuw.ac.in/](https://www.igdtuw.ac.in/)"> IGDTUW</a>
+<hr>
+
+List of tasks to be done:
+<ul>
+<li><strike>Task1 : Set up the project</strike></li>
```
Comment (soumyaa1804) [inline participants/Prerna-Goel.md:16]: See you forgot `<` in closing the li tag.

File: participants/Prerna-Goel.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+<li><strike>Task2 : Make your first PR</strike>/li>
```
Comment (soumyaa1804) [inline participants/Prerna-Goel.md:17]: See you forgot < in closing the li tag.

Comment (Prerna-Goel) [reply participants/Prerna-Goel.md:17]: okay...i wll correct it

Comment (Prerna-Goel) [reply participants/Prerna-Goel.md]: done changes

Comment (soumyaa1804) [reply participants/Prerna-Goel.md]: Great!

**Pull request merged**